### PR TITLE
Allow users to specify dbUser and dbPassword in config

### PR DESCRIPTION
**Pull request by mamikonyana**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +36,6 @@ bench/workflows/*
 
 # Annoying Mac file
 *.DS_Store
+
+# Arsen's local server configuration
+conf/arsen*
```
Comment (pbailis) [inline .gitignore]: Hmm, not sure we want this in the main source tree.

File: src/main/java/macrobase/runtime/MacroBaseServer.java [hunk 2]
```diff
@@ -46,6 +46,7 @@ public void run(ServerConfiguration configuration,
 
         if(configuration.doPreLoadData() && (!configuration.getPreLoad().isEmpty() || true)) {
             for(ServerConfiguration.PreLoadData pld : configuration.getPreLoad()) {
+                loader.setDatabaseCredentials(pld.getDbUser(), pld.getDbPassword());
```
Comment (pbailis) [inline src/main/java/macrobase/runtime/MacroBaseServer.java]: Does this still work if dbUser and dbPassword are not set?

File: .gitignore (hunk 1 above)
Comment (mamikonyana) [inline .gitignore]: where else? I just put this in the gitignore to have a clean working copy when changing my configuration

File: src/main/java/macrobase/runtime/MacroBaseServer.java (hunk 2 above)
Comment (mamikonyana) [inline src/main/java/macrobase/runtime/MacroBaseServer.java]: yes it does

File: .gitignore (hunk 1 above)
Comment (pbailis) [inline .gitignore]: What about just something like `conf/custom`? As it stands, everyone who downloads the code will have a special case for `conf/arsen`. :)

**Comment by mamikonyana** [comment]
I updated and now there is `conf/custom` in the .gitigrnoe

**Comment by pbailis** [comment]
Great, thanks! Feel free to merge via @deepakn94's workflow [on the wiki](https://github.com/stanford-futuredata/macrobase/wiki/Development-Notes#git-workflow) (pushing will close the PR).

**Comment by mamikonyana** [comment]
I just pushed to master, but nothing happened to this pull request. Should I force push to this branch too? That might also delete all the comments on the code.

**Comment by deepakn94** [comment]
Nope it's fine, no need to push to this branch, I'm just going to manually close this PR. (the reason it didn't close was because this local branch and the master branch diverged -- that is, the new master branch is not simply the old master branch + this branch on top of that)

**Comment by mamikonyana** [comment]
Right, but that would happen any time there is a rebase, no?

**Comment by deepakn94** [comment]
Right. I guess if you wanted to this the "right way", you would do what you alluded to earlier where you first force push to the remote branch, and then merge --ff-only on your local master branch before pushing.

**Pull request closed without merging**